### ISSUE TITLE
LUFA dependency for the usual usecase unclear

### ISSUE THREAD

**Issue by saphieron** [es-ude/BazelCProjectCreator#6]
It is undocumented and unclear what the LUFA dependency should be for. Should either be specified in the documentation together with how to use that dependency, or removed.

**Comment by glencoe** [comment]
Yep... don't like that dependency there either. What about making it optional for people that need it? We could introduce a flag like `-include_lufa`. Additionally documentation on how to use the lufa dependency should be included (but in https://github.com/es-ude/EmbeddedSystemsBuildScripts).
